controllers/configmap_controller.go

```diff
@@ -130,9 +124,26 @@ func NewConfigMapReconciler(mgr manager.Manager, clusterConfig cluster.Config, w
 		},
 		servicesManifest: svcData,
 		proxyEnabled:     proxyEnabled,
+		VXLANPort:        clusterConfig.Network().VXLANPort(),
 	}, nil
 }
 
+func generateServicesManifest(client client.Client, port string, platform oconfig.PlatformType) (*servicescm.Data, error) {
```

Comment:
For style reasons, please move helpers to the end of the file, not between the method declarations